`make build-in-docker` assumes that $PWD is mounted into minikube vm as $PWD

Second run output:

````
[james@bully:~/src/oaf/morph-ng/herokuish] for-morph-ng ± make build-in-docker
docker build --rm -f Dockerfile.build -t herokuish-build .
Sending build context to Docker daemon  81.92kB
Step 1/5 : FROM golang:1.12.0
 ---> cee68f119e19
Step 2/5 : ENV DEBIAN_FRONTEND noninteractive
 ---> Using cache
 ---> bdee1c547b07
Step 3/5 : RUN apt-get update -qq  && apt-get install -qq -y --force-yes      apt-transport-https      apt-utils      ca-certificates      curl      gnupg2      software-properties-common  && curl -fsSL https://download.docker.com/linux/$(. /etc/os-release; echo "$ID")/gpg | apt-key add -  && add-apt-repository    "deb [arch=amd64] https://download.docker.com/linux/$(. /etc/os-release; echo "$ID") $(lsb_release -cs) stable"                                                                               
 ---> Using cache
 ---> 630f1305ac17
Step 4/5 : RUN apt-get update -qq  && apt-get install -qq -y --force-yes docker-ce=17.12.0~ce-0~debian                                                                  
 ---> Using cache
 ---> 3e8ea7c32a15
Step 5/5 : WORKDIR /src
 ---> Using cache
 ---> 54b8f0dcd657
Successfully built 54b8f0dcd657
Successfully tagged herokuish-build:latest
docker run --rm -v /var/run/docker.sock:/var/run/docker.sock:ro \
        -v /var/lib/docker:/var/lib/docker \
        -v /home/james/src/oaf/morph-ng/herokuish:/src/github.com/gliderlabs/herokuish -w /src/github.com/gliderlabs/herokuish \                                        
        -e IMAGE_NAME=herokuish -e BUILD_TAG=dev -e VERSION=master \
        herokuish-build make -e deps build
make: *** No rule to make target 'deps'.  Stop.
make: *** [Makefile:39: build-in-docker] Error 2
````

--- a/README.md
+++ b/README.md
@@ -4,7 +4,10 @@
 
 #### Main dependencies
 
+**NB:** Later instructions will assume that your code checkouts are under `$HOME`. If this is not the case for you, you'll have to modify instructions to suit.
+
 * [Minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/).
+  *Use `minikube start --mount --mount-string $HOME:$HOME` to create the minikube cluster.*
 * [Skaffold](https://skaffold.dev/docs/getting-started/)
 * [jq](https://stedolan.github.io/jq/)
 * [MinIO client (mc)](https://min.io/download)

